src/components/inputs/ControlContainer/ControlContainer.scss

```diff
@@ -0,0 +1,27 @@
+$C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-control-container;
+
+.#{$C} {
+  &__error-container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
```

Comment:
shouldn't the error be aligned to the left?

Reply:
it is aligned to the left, center is for vertical alignment